src/core/reopt.jl

```diff
@@ -565,6 +574,7 @@ function add_variables!(m::JuMP.AbstractModel, p::REoptInputs)
 		dvPeakDemandMonth[p.months, 1:p.s.electric_tariff.n_monthly_demand_tiers] >= 0  # Peak electrical power demand during month m [kW]
 		MinChargeAdder >= 0
         binGHP[p.ghp_options], Bin  # Can be <= 1 if require_ghp_purchase=0, and is ==1 if require_ghp_purchase=1
+		dvBatteryIncluded[p.s.storage.types.all], Bin # 0 if no battery is included, 1 if battery is included
```

Comment:
Should this be indexed on `p.s.storage.types.elec` instead of all?

Reply:
Fixed to be p.s.storage.types.elec